src/App.tsx

```diff
@@ -1,14 +1,49 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = React.useState<Todo[]>([]);
+  const [loading, setLoading] = React.useState(true);
+  const [selectedTodo, setSelectedTodo] = React.useState<Todo | null>(null);
+  const [search, setSearch] = React.useState('');
+  const [filter, setFilter] = React.useState('all');
+
+  useEffect(() => {
+    setLoading(true);
+    getTodos().then(todosList => {
+      setTodos(todosList);
+      setLoading(false);
```

Comment:
you can move it to finally

Reply:
Did it, all tests passed locally, but not on GitHub.